### ISSUE TITLE
Teams are same and the team didn't win. 

### ISSUE THREAD

**Issue by xmyoard** [andrei1058/BedWars1058#179]
Teams are the same and the team didn't win. 
in this image the enemy team is in my team and we didn't even win and nothing happened tho 
![2021-12-08_16 59 38](https://user-images.githubusercontent.com/87631878/145205116-22ec3e80-837a-4092-a317-634dae8ccbed.png)

**Comment by reussy** [comment]
There is little information to understand the bug, please use the template and explain more


**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**System (please complete the following information):**
 - OS: [e.g. iOS]
 - Server software [e.g. spigot, paper]
 - Server Version [e.g. 1.8.8]
 - Plugin Version [e.g. 21.10.1]

**Additional context**
Add any other context about the problem here.

**Comment by ghost** [comment]
He is telling that When you join any arenaGroup other than Solo and have minimum player to start to 2 then both the players are in the same team instead of balancing the team to like red and blue

**Comment by ghost** [comment]
I have the same problem with all the server version.
1.8-1.16
Please fix it.
Also if you do /bw start debug with 2 players in The Doubles 3v3v3v3 4v4v4v4 then you get into same team instead of being seperate teams

**Comment by ajgeiss0702** [comment]
Disable the experimental team assigner in the config

**Comment by xmyoard** [comment]
still having issues

**Comment by ajgeiss0702** [comment]
> still having issues

Did you disable the experimental team assigner?

**Comment by xmyoard** [comment]
Yes, when i do that the problem doesn't gets loved and when we try to leave it says "Victory" and the whole server restarts :/

**Comment by tomkeuper** [comment]
Is this issue still relevant? I have seen some similair issues regarding teams not being balanced properly but that doesnt really seem the case here. 

Might also have something to do with using /bw start debug because this prevents the game from ending with just one team. Thats why debug is only used for... debugging :)

**Comment by xmyoard** [comment]
I'm sorry, I don't use this plugin anymore.

**Comment by YukiEnd233** [comment]
> 

Just enable "experimental team assignment" in config and start the game normally
and experimental team assignment is true by default

**Comment by EndMoyu** [comment]
> Is this issue still relevant? I have seen some similair issues regarding teams not being balanced properly but that doesnt really seem the case here.
> 
> Might also have something to do with using /bw start debug because this prevents the game from ending with just one team. Thats why debug is only used for... debugging :)

Just enable "experimental team assignment" in config and start the game normally
and experimental team assignment is true by default

**Comment by xiaozhu52055** [comment]
> 
experimental team assignment
It doesn't work
